requirements.txt

```
@@ -1,6 +1,6 @@
 contextvars>=2.4;python_version<'3.7'
 jsonschema>=2.6
-retworkx>=0.8.0
+retworkx @ git+https://github.com/Qiskit/retworkx@main
```

retworkx 0.9.0 has been released now so we can update this:

```suggestion
retworkx>=0.9.0
```